tasmota/tasmota_support/support.ino

```diff
@@ -590,6 +590,33 @@ char* Trim(char* p) {
   return p;
 }
 
+size_t HexStringToBinaryString(char** source) {
+  size_t str_len = strlen(*source+2);
+  char *s = *source+2;
```

Comment:
I'm not very comfortable with just skipping blindly the first 2 characters. Could you at least check if they are "0x" or "0X" and skip them in such case, or keep them if it's just HEX data without '0x'?

Reply:
Changed the format to no longer require 0x prefix.